src/main/java/org/sagebionetworks/web/client/widget/entity/controller/EntityActionControllerImpl.java

```diff
@@ -438,9 +440,8 @@ public void configure(ActionMenuWidget actionMenu, EntityBundle entityBundle, bo
 		this.entityTypeDisplay = getFriendlyEntityTypeName(entityBundle.getEntity());
 		this.currentArea = currentArea;
 
-		// hide all commands by default
-		actionMenu.hideAllActions();
-		gwt.scheduleExecution(reconfigureActionsCallback, 2000);
```

Comment:
What was the original purpose of this delay?

Reply:
No idea.  It was a change I made 4 years ago alongside a number of performance improvements:
https://github.com/Sage-Bionetworks/SynapseWebClient/commit/6fe87966e115028de586eb9002f96327f51191b8

I'm very please if it can be deleted!